server/views/pages/contacts/manage/contactConfirmation/identityNumbers.njk

```diff
@@ -11,7 +11,7 @@
                         text: (item.identityTypeDescription | capitalizeFirstLetter)
                     },
                     value: { 
-                        html: item.identityValue + ('<p>' + item.issuingAuthority + '</p>' if item.identityType == 'PASS' else ''),
+                        html: item.identityValue + ('<p>Issuing authority- ' + item.issuingAuthority + '</p>' if item.identityType == 'PASS' else ''),
```

Comment:
I think there should be a space after Issuing authority and before -

Reply:
In local:
![image](https://github.com/user-attachments/assets/2f149e31-ca44-418b-8398-7b0c533d3891)

AC:
![image](https://github.com/user-attachments/assets/a8d62462-4f06-4956-81f9-747081d612e0)